src/components/ImageGeneration/MarkerFiltersDropdown.tsx

```diff
@@ -120,7 +122,8 @@ export function DumbMarkerFiltersDropdown({
                   setMarker(checked ? marker : undefined);
                   setFilters({ marker: checked ? marker : undefined });
                 }}
-                {...chipProps}>
+                {...chipProps}
+              >
                 <Group spacing={4} noWrap>
```

Comment:
Doesn't need `Group`. Could simply be a `div`. Do we even need the `div`?

Reply:
the whole section `const dropdown = ...` could be refactored to use tailwind. There is an extra, unnecessary `Stack` wrapping your content.

Reply:
Agreed, we are hoping to start migrating away from CSS-in-JS as much as possible, if possible use tailwind classes to layout your components